### ISSUE TITLE
default hotkey overrides goto_symbol_in_project command

### ISSUE THREAD

**Issue by nicck** [wulftone/sublime-text-quick-file-move#7]
even if alternative hotkey for quick_file_move is present in user keymap.

workaround:

``` js
  // new hotkeys for quick_file_move
  { "keys": ["ctrl+shift+r"], "command": "quick_file_move" },

  // return default command for super+shift+r
  { "keys": ["super+shift+r"], "command": "goto_symbol_in_project" }
```

**Comment by wulftone** [comment]
Thanks for the info.  I didn't know ST3 had claimed that key combo.  I've changed it to `alt+shift+r` (some computers don't have a `super` key).

Closed via f2ca3795e0dfd97285b81f4e549c7af3999fd1da

**Comment by nicck** [comment]
@wulftone I'm not sure `ctrl+shift+r` is used by default in ST3 on Windows and Linux, but `super+shift+r` on OSX is. We probably need to update this `Default (OSX).sublime-keymap` instead.

**Comment by wulftone** [comment]
Oops, sorry, I read your initial description in dyslexia-mode.

This is from my (Linux) default keymap in ST3:

```
{ "keys": ["ctrl+shift+r"], "command": "goto_symbol_in_project" },
```

I don't use a Mac, so I'll change it the way you describe, unless you think `alt` would be better...?  Do Macs even have an `alt` key?

**Comment by nicck** [comment]
Sure, `alt` is present on [mac keyboard](http://d.pr/i/H62u).
Looks like in default keymap for OSX in most cases `ctrl` replaced by `super` (aka meta or cmd).
So, I guess `alt+shift+r` is good choice for all platforms.

**Comment by wulftone** [comment]
Okay, closed via ee48d0bfeda9a77d084af71fe76b6e06590dee6c

Thanks!
